### PR TITLE
add phixtral and optimize phi-moe

### PR DESCRIPTION
**Pull request by ivy-lv11**
Add phixtral-4x2_8 example and optimize phi-moe. The performance could be found in the issue https://github.com/analytics-zoo/nano/issues/988 . iGPU is also verified.

### CODE REVIEW

**Comment by shane-huang** [comment]
Update the tables in python/llm/README.md and BigDL/README.md

File: python/llm/example/GPU/PyTorch-Models/Model/phixtral/generate.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+import torch
+import time
+import argparse
+import numpy as np
+
+from transformers import AutoTokenizer, GenerationConfig
+import intel_extension_for_pytorch as ipex
+print(ipex.__version__)
+from torch.profiler import profile, record_function, ProfilerActivity
```
Comment (plusbang) [inline python/llm/example/GPU/PyTorch-Models/Model/phixtral/generate.py:25]: I think we could remove L24 and L25

Comment (ivy-lv11) [reply python/llm/example/GPU/PyTorch-Models/Model/phixtral/generate.py]: Sure. Already removed.

File: python/llm/example/CPU/HF-Transformers-AutoModels/Model/phixtral/README.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+<|Bot|>:
+-------------------- Output --------------------
+<|User|>:AI是什么？
+<|Bot|>:AI是人工智能的缩写，是计算机科学的一个分支，旨在使计算机能够像人类一样思考、学习和执行任务。AI技术包括机器学习、自然
+```
+
+```log
+Inference time: xxxx s
+-------------------- Prompt --------------------
+<|User|>:What is AI?
```
Comment (plusbang) [inline python/llm/example/CPU/HF-Transformers-AutoModels/Model/phixtral/README.md:76]: The sample output seems to mismatch the prompt template in generate.py, please confirm it.

**Review by plusbang — APPROVED** [review]
LGTM

**Pull request merged**